File: Questions in Java/1020.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws Exception {

        Scanner input = new Scanner(System.in);//--Entrada de Dados

       // Declaração de variavel 

        int Dias = input.nextInt();
        input.close();
       // Atribuições das variaveis

        int Ano = Dias/365;
        int mes = ((Dias%365)/30);
        int Dia = ((Dias%365)%30);

       // Saida de Dados
       
       System.out.println(Ano +" ano(s)");
       System.out.println(mes +" mes(es)");
       System.out.println(Dia +" dia(s)");
    }
}

File: Question-in-Python/1020.py
X = int(input()) #--Entrada de dados

A = X//365 #--Quantidade de Anos
R = X%365 #--Restante dos dias
M = R//30 #--Quantidade de meses
D = R%30 #--O restante de dias

print(f'{A} ano(s)\n{M} mes(es)\n{D} dia(s)')